source/release-notes/backend.txt

```diff
@@ -13,6 +13,35 @@ Atlas App Services Changelog
 2023 Releases
 -------------
 
+.. _backend_20231213:
+
+13 December 2023 Release
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Added support for passing in user's new metadata from a request using JWT authentication to the function context.
+
+- Apps, users, and refresh token are now pruned at the 30 day mark.
+
+- Fixed webpack production yarn build warnings, such as "asset size limit" and "entrypoint size warnings", by reducing bundle size. 
+
+- Updated App Services schema UI to display the difference between a 64-bit ``long`` and 32-bit ``integer``.
+
+- Disallowed creating schemas with invalid database or collection names, such as names including spaces.
+
+- Left default for ``allowDiskUse`` as true unless user specifies it as false. 
```

Comment:
```suggestion
- MongoDB aggregation operations no longer specify a default value for ``allowDiskUse``.
```

If you check out [the code in the PR](https://github.com/10gen/baas/pull/12029/files#diff-fa3d6d98d704671b1cd8b1ed6516651388cd87db0dbb61dd8cb3eecbf36140abR238-R239) it's only setting the value if the user passes it.